src/wasm/wasm-binary.cpp

```diff
@@ -2967,7 +2972,7 @@ void WasmBinaryReader::readImports() {
         wasm.addGlobal(std::move(curr));
         break;
       }
-      case ExternalKind::Tag: {
+      case (uint32_t)ExternalKind::Tag: {
```

Comment:
(though, is the typedef type worth it, if it forces these annotations?)

Reply:
Because we use an `enum class`, the enum values are not implicitly convertible to uint32_t, which is really annoying when we need to handle the extra bit or'd in.